### PR TITLE
Show fullname on issue edits and gpg/ssh signing info

### PR DESCRIPTION
**Pull request by 42wim**
Show missing full names when configured to do so

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by singuliere — APPROVED** [review]
Well done.

File: routers/web/repo/issue_content_history.go [hunk 1]
```diff
@@ -68,9 +69,15 @@ func GetContentHistoryList(ctx *context.Context) {
 			actionText = ctx.Locale.Tr("repo.issues.content_history.edited")
 		}
 		timeSinceText := timeutil.TimeSinceUnix(item.EditedUnix, lang)
+
+		username := item.UserName
+		if setting.UI.DefaultShowFullName {
+			username = item.UserFullName
+		}
```
Comment (zeripath) [inline routers/web/repo/issue_content_history.go]: Don't you also need to check that the full name is not ""?

```suggestion
		if setting.UI.DefaultShowFullName && strings.TrimSpace(item.UserFullName) != "" {
			username = strings.TrimSpace(item.UserFullName)
		}
```

Comment (42wim) [reply routers/web/repo/issue_content_history.go]: yes, good catch

**Comment by lunny** [comment]
Should we really need to backport this to v1.16?

**Comment by 42wim** [comment]
@lunny why not? it's a simple bugfix?

**Comment by lunny** [comment]
> @lunny why not? it's a simple bugfix?

Could you paste some screenshots?

**Comment by 42wim** [comment]
before
image

after
image

image

**Comment by zeripath** [comment]
Please send backport

**Comment by 42wim** [comment]
@zeripath #18828

**Pull request merged**